src/constants.js

```diff
@@ -39,6 +39,9 @@ export const NETWORK_CONCURRENCY = 8;
 // HTTP timeout used when downloading packages
 export const NETWORK_TIMEOUT = 30 * 1000; // in milliseconds
 
+// Delay before retrying HTTP requests after failure
+export const RETRY_DELAY = 3 * 1000; // in milliseconds
```

Comment:
I know this is how `NEWWORK_TIMEOUT` is setup, but I think a flag that could provide an override would be nice here. Some CI's may require different retry intervals.